dist/placeholder-buster.txt

```diff
@@ -51,6 +51,20 @@ dailymotion.com##div[class^="AdDiscovery"]
 # https://github.com/NanoAdblockerLab/NanoContrib/pull/16#issuecomment-473483169
 myspace.com##.container-main > ul > li:has(div[id="ad-placement"])
 
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/17
+!#if env_mobile
+instagram.com##footer:style(margin-top: 25px !important)
+!#endif
```

Comment:
Hum... Looks like more of a bug of the website. Usually we don't want to try to fix other people's bugs.

Reply:
Duly noted.